modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -310,6 +309,99 @@ else if (clientExtension.type.equals("themeCSS")) {
 			createClientExtensionConfigTaskProvider, workspaceExtension);
 
 		_configureLiferayRoutes(project, workspaceExtension);
+
+		if (_isSpecialLanguageProject(project)) {
+			TaskProvider<GenerateLanguageBatchConfigTask>
+				generateLanguageBatchConfigTaskProvider =
+					GradleUtil.addTaskProvider(
+						project, GENERATE_LANGUAGE_BATCH_TASK_NAME,
+						GenerateLanguageBatchConfigTask.class);
```

Comment:
Generally, for tasks like this we should use a two-step structure of:
1. Call `_createGenerateLanguageBatchConfigTaskProvider` that creates the task and adds any necessary metadata (group, description, etc).
2. Call `_configureGenerateLangaugeBatchConfigTask` that configures the task with dependencies and any inputs.